lib/kdump_utils.pm

```diff
@@ -29,7 +29,7 @@ sub prepare_for_kdump_sle {
         my $sles_debug_repo = get_var('REPO_0') =~ s/(Server|Desktop)/\U$1\E/r;
         $sles_debug_repo =~ s/DVD/POOL/;
         $sles_debug_repo =~ s/Media1/Media3/;
-        my $url = "ftp://openqa.suse.de/$sles_debug_repo";
+        my $url = "$utils::OPENQA_FTP_URL/$sles_debug_repo";
```

Comment:
This will work for production, but for local instances variable is set to localhost. So I'll keep this as it is.